sqrl-tools/sqrl-packager/src/test/resources/package-configtest.json

```diff
@@ -6,7 +6,7 @@
     "version": "1.0.0",
     "variant": "dev",
     "description": "Basic nutshop example",
-    "keywords": ["example", "tutorial", "ecommerce"]
+    "topics": ["example", "tutorial", "ecommerce"]
```

Comment:
There are a lot of other package.json's that have "keywords" instead of topics. Will you migrate them here too?

Reply:
Hmm that's strange. Could you point me to one? This is what I see:
![image](https://github.com/DataSQRL/sqrl/assets/11146937/2864e344-2fa1-4f44-8ebc-26a39c639295)

Reply:
Ah sorry, I needed to do a git clean.